src/javaharness/java/arcs/api/ParticleImpl.java

```diff
@@ -6,6 +6,7 @@
 
 public class ParticleImpl implements Particle {
```

Comment:
Probably better to call this ParticleBase if it is meant to be a base class extended by actual builtin particles.

Reply:
it is, renamed.